package.json

```diff
@@ -45,7 +45,9 @@
     "@types/react-window": "1.8.x",
     "@types/react-window-infinite-loader": "1.0.x",
     "@types/sinon": "10.x",
+    "@types/streamsaver": "^2.0.4",
     "@types/string-natural-compare": "3.0.x",
+    "@types/xml2js": "^0.4.14",
```

Comment:
resolved in [027bc6d](https://github.com/AllenInstitute/biofile-finder/pull/204/commits/027bc6dd1c70892b7b14a5f81606886c1f5dd76f) where it is removed since we dont use xml2js anymore.